kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -125,19 +133,39 @@ class ResidualBasedBlockBuilderAndSolver
         // Validate default parameters
         Parameters default_parameters = Parameters(R"(
         {
-            "name" : "ResidualBasedBlockBuilderAndSolver"
+            "name"                               : "ResidualBasedBlockBuilderAndSolver",
+            "scale_diagonal"                     : false,

```

Comment:
as i was telling in my comment, i don't like having multiple parameters to control this. 

what is the difference in meaning between "scale_diagonal" and "diagonal_values_for_dirichlet_dofs"?

Reply:
well, I can add an option of "non_scale" and consider it by default

Reply:
i do not understand what you want to achieve by "scale_diagonal"